AirCasting/Settings/MyAccountViewSignOut.swift

```diff
@@ -32,11 +32,12 @@ private extension MyAccountViewSignOut {
     
     var signOutButton: some View {
         Button(action: {
-            do {
-                try logoutController.logout()
-            } catch {
-                assertionFailure("Failed to deauthorize \(error)")
-            }
+            fatalError("CRASHLYTICS TEST")
+//            do {
+//                try logoutController.logout()
+//            } catch {
+//                assertionFailure("Failed to deauthorize \(error)")
+//            }
```

Comment:
Same as above - commented code, why? :D

Reply:
same: I cannot see it, maybe from earlier commit? 🙁

Reply:
Weird, but true, the code is different from what github is suggesting it is.